library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/utils/MiscUtilsTest.kt

```diff
@@ -118,14 +118,14 @@ internal class MiscUtilsTest {
             whenever(it.configuration).thenReturn(fakeConfiguration)
             whenever(it.displayMetrics).thenReturn(fakeDisplayMetrics)
         }
-        val mockActivity: Activity = mock {
-            whenever(it.windowManager).thenReturn(mockWindowManager)
+        val mockContext: Context = mock {
```

Comment:
minor: There are lines of code in this class that repeat themselves in several tests. E.g. mockContext, mockResources, fakeConfiguration, fakeDisplayMetrics, etc. We could refactor to reduce this